packages/manager/apps/hpc-vmware-managed-vcd/src/components/datagrid/compute/ComputeOrderCells.component.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { Dispatch, SetStateAction, useContext } from 'react';
+import { useTranslation } from 'react-i18next';
+import {
+  DataGridTextCell,
+  Description,
+  OvhSubsidiary,
+  Price,
+} from '@ovh-ux/manager-react-components';
+import { ShellContext } from '@ovh-ux/manager-react-shell-client';
+import { OsdsRadio, OsdsRadioButton } from '@ovhcloud/ods-components/react';
+import { ODS_RADIO_BUTTON_SIZE } from '@ovhcloud/ods-components';
+import { ODS_THEME_COLOR_INTENT } from '@ovhcloud/ods-common-theming';
+import { IVdcOrderableVhostPriced } from '@/types/vcd-vdc-orderable-resource.interface';
+
+export const ComputeOrderSelectCell = ({
+  vHost,
+  selectedVhost,
+  setSelectedVhost,
+}: {
+  vHost: IVdcOrderableVhostPriced;
+  selectedVhost: string;
+  setSelectedVhost: Dispatch<SetStateAction<string>>;
+}) => (
+  <DataGridTextCell>
+    <OsdsRadio
+      checked={vHost?.profile === selectedVhost}
```

Comment:
```suggestion
      checked={vHost.profile === selectedVhost}
```
Is the optional chaining really necessary on the datagrid cells?

Reply:
No you're right 👍